parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ParquetFileReader.java

```diff
@@ -771,7 +787,7 @@ DictionaryPage readDictionary(ColumnChunkMetaData meta) throws IOException {
         compressedPage.getEncoding());
   }
```

Comment:
we should open a JIRA to factor out those constructor and clean up the deprecated ones.
lots of duplication here.